axelor-base/src/main/java/com/axelor/apps/base/service/administration/SequenceService.java

```diff
@@ -67,7 +67,9 @@ public class SequenceService {
   protected static final String PATTERN_DAY = "%D";
   protected static final String PATTERN_WEEK = "%WY";
   protected static final String PADDING_STRING = "0";
```

Comment:
```suggestion
  protected static final String PADDING_DIGIT = "0";
```
I added these 2 last suggestions only to follow common good practice and to have more meaningful names. But otherwise, good job!